plugins/widget/plugin.js

```diff
@@ -1939,7 +1939,7 @@
 				enterMode: this.enterMode
 			} );
 
-			this.setHtml( data );
+			this.setHtml( CKEDITOR.tools.htmlSafeByReview(data, 'data goes throguh multiple different dataProcessor stages to get rid of any unsafe code.') );
```

Comment:
Maybe we should legacy convert here because we can't see the multiple stages at a quick glance, and that logic before this point may change.

Reply:
Related to comment on tableselection/plugin.js

Reply:
Moved to PR # 11. Legacy converted

Reply:
should we remove this change here if you made this a legacy conversion?

Reply:
Yep just removed it, thanks!